### ISSUE TITLE
Declare as "stable"

### ISSUE THREAD

**Issue by goerz** [JuliaDocs/IOCapture.jl#24]
This package feels pretty "stable" to me, so it should probably be tagged as 1.0 to indicate that.

In my opinion, `v1.0` releases shouldn't be breaking: A `v1.0` release should be mostly identical to the last `v0.x.y` release and just serve as an indication that "the API is now stable".

So, if it were up to me, I'd merge/release the recent #20 and #23 as `v0.2.4` and then release `v1.0` some weeks/months after that release has proven not to break anything in the wider ecosystem.

Of course, there are different opinions on the correct interpretation of SemVer, so make your own decisions ;-)

I just wanted to put the idea out there that this package seems to be mature enough to have a `v1.0` label.

**Comment by MichaelHatherly** [comment]
That's reasonable, similar approach I've taken with personal packages recently as well. There's always `v2` if something needs breaking in the future anyway.

**Comment by mortenpi** [comment]
Yep, agreed, I've been thinking along the same lines for a while. Waiting a few weeks before tagging 1.0.0, just in case, also seems prudent.

**Comment by goerz** [comment]
Great! Do you feel like tagging `v0.2.4`? I'd love to use my fancy new feature ;-)

**Comment by mortenpi** [comment]
Yep, working on that :slightly_smiling_face:

**Comment by goerz** [comment]
With @fingolfin sprucing up the documentation, maybe the next release will be a good opportunity to move to a `v1.0.0` version number

**Comment by mortenpi** [comment]
I think let's just do it. We haven't had any changes for more than a year, and there's nothing obvious on the horizon either.
